packages/@okta/vuepress-theme-prose/assets/css/components/_stackSelector.scss

```diff
@@ -61,61 +68,95 @@ $small-spacing: 0.75rem;
       text-overflow: ellipsis;
     }
 
-    .vs__search {
-      margin-top: 0;
-      margin-left: 1.7rem;
-    }
-
-    .vs__dropdown-toggle{
-      min-height: 45px;
-      padding: 0.5rem;
-      display: flex;
-      align-items: center;
-      justify-content: space-between;
-      flex-wrap: nowrap;
-      .vs__selected-options{
-        flex-basis: auto;
-        padding: 0;
-        padding-left: 0.5rem;
-        max-width: 95%;
-        .vs__selected{
-          margin: 0;
-          padding: 0;
-          border: 0;
-        }
+    .v-select{
+      .vs__dropdown-toggle{
+            border-color: $dropdown-toggle-border;
+            border-radius: 4px;
+            height: 38px;
+            padding: 0 16px;
+            display: flex;
+            align-items: center;
+            justify-content: space-between;
+            flex-wrap: nowrap;
+            .vs__open-indicator{
+              fill: cv('gray', '900');
+            }
+            .vs__search {
+              height: 0;
+              width: 0;
+              margin: 0;
+              padding: 0;
+            }
+            .vs__selected{
+              border: none;
+              padding: 0;
+              margin: 0;
+              position: relative;
+            }
+            .vs__selected-options{
+              flex-basis: auto;
+            }
       }
-    }
 
-    .vs__dropdown-menu {
-      overflow-x: hidden;
-      border-top-style: initial;
-      padding-bottom: 0px;
-      z-index: 50;
+      .vs__dropdown-menu {
+            border: 1px solid #1662de;
```

Comment:
Would the existing [color constant](https://github.com/okta/okta-developer-docs/blob/master/packages/%40okta/vuepress-theme-prose/assets/css/abstracts/_colors.scss#L19) be appropriate here?